### DISCUSSION TITLE
Isolating system effects from gameplay effects?

### DISCUSSION THREAD

**Discussion by Crozh** [ppy/osu#17439] [Ideas] [answered]
I'm having this battle were I have to balance the effects options because I can't hear some in-game effects due to how 'unbalanced' skin's hit-sounds are. But at the same time if I want turn up the effects option I'll get deaf from some of the menu sounds.  
I don't know if this has been suggested before, or if this is even a problem for some people, but for me its getting kind of annoying.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/14529

**Reply by Crozh** [reply]
Figured this was suggested, thanks.
